mungegithub/mungers/submit-queue_test.go

```diff
@@ -86,9 +86,15 @@ func BareIssue() *github.Issue {
 func LGTMIssue() *github.Issue {
 	return github_test.Issue(someUserName, 1, []string{claYesLabel, lgtmLabel}, true)
 }
+func LGTMApprovedIssue() *github.Issue {
+	return github_test.Issue(someUserName, 1, []string{claYesLabel, lgtmLabel, approvedLabel}, true)
+}
 
+func OnlyApprovedIssue() *github.Issue {
+	return github_test.Issue(someUserName, 1, []string{claYesLabel, approvedLabel}, true)
+}
 func DoNotMergeIssue() *github.Issue {
```

Comment:
@calebamiles does this work?  It's used below and should NOT lead to status merged.  Should be pending with the message that there's no lgtm or approved.